### DISCUSSION TITLE
Do we need SQLite for OCPP client

### DISCUSSION THREAD

**Discussion by Bhuvi-Naveen** [c-jimenez/open-ocpp#125] [General]
Hi, 
Thanks to reply.
I would like to know, do we require SQLite for OCPP client.

**Comment by c-jimenez** [comment]
Hi,

Yes SQlite is needed since persistency for offline messages, charging profiles et connector status is made through SQLite database.

**Comment by Bhuvi-Naveen** [comment]
Thank you c-jimenez
